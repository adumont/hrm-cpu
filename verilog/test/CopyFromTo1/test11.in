43 ff
12 85
58 70
07 52
57 ce
37 da
41 60
34 9b
01 f5
06 06
48 0d
21 aa
12 c0
03 d7
11 51
23 02
48 19
56 73
23 16
20 8e
15 67
44 35
34 70
45 c3
41 85
15 a8
16 99
38 f9
36 8f
36 9a
24 76
08 1e
02 b7
13 c9
42 22
35 15
49 f5
11 ac
49 05
27 da
05 ab
49 9d
53 61
47 10
41 1f
49 de
27 80
12 82
40 0c
