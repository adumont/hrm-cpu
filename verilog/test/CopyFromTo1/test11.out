ff
85
70
52
ce
da
60
9b
f5
06
0d
aa
c0
d7
51
02
19
73
16
8e
67
35
70
c3
85
a8
99
f9
8f
9a
76
1e
b7
c9
22
15
f5
ac
05
da
